docs/chapters/architecture/SOTA-system.rst

```diff
@@ -617,13 +617,66 @@ see the status of your rollout campaign changed.
 Going further
 ^^^^^^^^^^^^^
 
-Those instructions don't use a database to store artifacts. This means that
-every time Hawkbit will be restarted, its rollout campaigns will be lost. This
-is handy for a development environment but unsustainable for a real world
-scenario. Please refer to `the Hawkbit documentation`_ for details on how to set it
-up with a MariaDB database.
+Persistent storage
+""""""""""""""""""
+
+Those instructions don't use a database to store artifacts and metadata. This
```

Comment:
I know this is an old paragraph, but I think this might be a non-English way of referring to something. I don't understand what instructions are referenced here. "Those" implies to me that they were just mentioned (but there is no text above), but I would anyway have used "These" or "The following" or if referenced earlier "The above mentioned"